config/runtime.exs

```diff
@@ -407,6 +407,16 @@ end
 
 ##### PROXY REPO CONFIGURATIONS #####
 
+poolboy_config = [
```

Comment:
we can rename this to bbc_parser_config

it's the poolboy config for a specific part of the code;
in the case we add another poolboy managed process, we'll need the extra namespace